base_station/gui/src/components/SASiteControls.vue

```diff
@@ -89,11 +91,155 @@
           'type': 'MicroCam',
           'id': 'camera_' + this.site
         })
+      },
+      swapComponent: function(component)
+      {
+        this.currentComponent = component;
       }
     },
 
     components: {
-      Checkbox
+      Checkbox,
+      'Chlorophyll': {
+        template: '<div><p>Turn off White LEDs</p><button id="execute" @click="swapComponent(\'WhiteLEDsOff\')">Execute</button></div>',
+        methods: {
+          swapComponent: function(component)
+          {
+            this.$parent.$parent.$parent.publish("/mosfet_cmd", {
+              'type': 'MosfetCmd',
+              'device': 2,
+              'enable': false
+            })
+            var site_letter = 'b';
+            if (this.$parent.site == "White") {
+              site_letter = 'w';
+            }
+            else if (this.$parent.site == "Yellow") {
+              site_letter = 'y';
+            }
```

Comment:
sorry what is white yellow blue for? this sounds like last years' science boxes

Reply:
Now that I look at it the SASiteControls vue component does not get used at all in the actual SATask.vue code. It is commented out and is now just a remnant of the old GUI prior to the redesign. Maybe we should clean up the components folder a bit.

Reply:
This should be resolved now